misc/Dockerfile.2

```diff
@@ -0,0 +1,9 @@
+FROM python:2
```

Comment:
Are you planning on committing these? If so, it should probably be its own Docker-related PR.